.ebextensions/cloudwatchlogs-apache/cwl-setup.config

```diff
@@ -1,138 +0,0 @@
-########################################################################
```

Comment:
The docs around this is very wooly but I have tested and still getting application logs after deploying on this branch